guides/common/modules/proc_enabling-the-flatpak-remote.adoc

```diff
@@ -0,0 +1,71 @@
+[id="enabling-the-flatpak-remote"]
+= Enabling the Flatpak remote
+
+This Hammer CLI procedure enables and manages Flatpak repositories.
+
+.Prerequisites
+* *view_flatpak_remotes*: The user can view Flatpak remotes.
+* *create_flatpak_remotes*: The user can create new Flatpak remotes.
+* *edit_flatpak_remotes*: Grants the user permission to change existing Flatpak remotes.
+* *destroy_flatpak_remotes*: The user can delete Flatpak remotes.
```

Comment:
This currently misses an introduction: What are you listing? The answer is permissions and it would be good to state that explicitly. I searched for other modules that include user permissions as prerequisites and this is a nice example: https://github.com/theforeman/foreman-documentation/blob/cf0c81548b54ee2480395210e96fcd9d2889f838/guides/common/modules/proc_customizing-host-registration-by-using-snippets.adoc?plain=1#L17
Note the markup and the fact that we don't typically explain what each permission does. (It's actually pretty self-explanatory.)